foundations/introduction/motivation_and_mindset.md

```diff
@@ -161,6 +165,7 @@ Learning any new skill is a journey full of speed bumps and obstacles to be over
   - [Believe you can get better](https://www.ted.com/talks/carol_dweck_the_power_of_believing_that_you_can_improve)
```

Comment:
The linter error here is confusing, but it should be fixed if you indent these bullets one more step to the right.